### PR TITLE
Add support for UMD module export.

### LINKED ISSUE #3: A mode without webpack.
We'd like to be able to use the library without the need to setup webpack.

**manifest.json**

```
  "background": {
    "scripts": [
      "vendor/testpilot-qa.js",
      "background.js"
    ]
  },
```

But when we do that, we have the exports is not defined error.

### PR DESCRIPTION
**Pull request by Natim**
Fixes #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,14 +9,16 @@
...
     "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-es2015-modules-umd": "^6.24.1",
     "babel-preset-env": "^1.6.0",
     "husky": "^0.14.3",
     "lint-staged": "^4.0.2",
     "prettier": "^1.5.3"
   },
   "scripts": {
-    "build": "mkdir -p dist && babel src/index.js --out-file dist/index.js",
+    "build": "mkdir -p dist && babel src/TestPilotGA.js --out-file dist/index.js",
```
Comment (chuckharmston) [inline package.json]: Minor thing, but mind doing `testpilot-ga.js` instead? Matches the NPM name.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "testpilot-ga",
-  "version": "0.1.2",
+  "version": "0.2.0",
```
Comment (chuckharmston) [inline package.json]: Could you also bump the version number in the example's dependencies?

File: package.json (hunk 1 above)
Comment (Natim) [inline package.json]: It has to match the export name reason why I had to change it in the first place.

Comment (chuckharmston) [inline package.json]: Ah ha! Works for me.

**Pull request merged**